javascript/javascript2/week1/øfunctionApp.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <script>
+       const danishWords = ["billie", "plante", "kaffe", "bog", "ø", "planetarium"];
+
+function getShortest() {
+  let shortest = danishWords.sort((a, b) => a.length - b.length)[0];
+  return shortest;
+}
+
+console.log(getShortest());
```

Comment:
this works, however, on very large arrays. a for loop would be a faster algorithm than using sort.

Reply:
thanks,.for large array, i will use loop , great